kolibri_zim_plugin/assets/src/views/ZimContentView.vue

```diff
@@ -191,13 +170,15 @@
     width: 100%;
     height: 100%;
     overflow: auto;
+    opacity: 1;
+    transition: opacity 0.5s ease;
     // Set the transform property so fixed-position children are relative to
     // this element.
     transform: translate(0);
   }
 
-  .load-content {
-    display: none;
+  .content--loading {
```

Comment:
Nit why 2 dashes in this class name? We don't have other cases in our source code.